gnomad_qc/v4/annotations/generate_freq.py

```diff
@@ -466,35 +491,54 @@ def combine_freq_hts(
     return freq_ht
 
 
-# Functions to correct frequencies and hists for high ab hets.
-def create_high_ab_age_hists_expr(ht: hl.Table, age_group_key="sample_age_bin"):
+def correct_for_high_ab_hets(ht: hl.Table, af_threshold: float = 0.01) -> hl.Table:
     """
-    Create histograms of high ab counts using age bins to account for high AB hets becoming hom alts.
+    Add documentation.
 
-    :param ht: Hail Table containing age hists, AB annotation.
-    :param age_group_key: Age group key to use for age histogram.
-    :return: Hail struct containing age histogram of high ab counts.
+    :param ht:
+    :param af_threshold:
+    :return:
     """
-    non_range_entries = hl.set(["n_larger", "n_smaller"])
-    age_bins_indices = hl.sorted(
-        hl.enumerate(ht["freq_meta"], index_first=False)
-        .filter(lambda x: x[0].contains(age_group_key))
-        .map(lambda x: (x[0][age_group_key], x[1]))
+    # TODO: Comment
+    call_stats_expr = hl.map(
+        lambda f, g: hl.struct(
+            AC=hl.int32(f.AC + g),
+            AN=f.AN,
+            homozygote_count=f.homozygote_count + g,
+            AF=hl.if_else(f.AN > 0, (f.AC + g) / f.AN, hl.missing(hl.tfloat64)),
```

Comment:
```suggestion
            AC=hl.int32(f.AC + g),
            AF=hl.if_else(f.AN > 0, (f.AC + g) / f.AN, hl.missing(hl.tfloat64)),
            AN=f.AN,
            homozygote_count=f.homozygote_count + g,
```
Changing the order so the if_else works on L529

Reply:
Thank you, I forgot to change after I put in the fix to gnomad_methods last week